about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone</title>
+    <link rel="stylesheet" href="assets/css/style.css" />
+
+    <!-- Font Awesome CDN -->
+    <script
+      src="https://kit.fontawesome.com/c26b883059.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <nav>
+      <input id="nav-toggle" type="checkbox" />
+      <div class="logo">
+        <a href="index.html"><strong>LOGO</strong></a>
+      </div>
+      <ul class="links">
+        <li><a class="active" href="about.html">About</a></li>
+        <li><a href="index.html">Program</a></li>
+        <li><a href="#join">Join</a></li>
+        <li><a href="#sponsor">Sponsor</a></li>
+        <li><a href="#news">News</a></li>
+        <li class="nav-item">
+          <a class="join-nav-button bd-primary" href="#campaign">CC Campaign</a>
+        </li>
+      </ul>
+      <div class="icon-burger nav-toggle">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+      </div>
+    </nav>
+
+    <div class="container">
+      <div class="row">
+        <div class="headline text-center">
+          <h1 class="h-1 color-primary">
+            CREATIVE COMMONS <br />
+            GLOBAL SUMMIT <br />
+            2015
+          </h1>
+          <p class="text-center box text-about">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore magnam eos repellendus eligendi sint quaerat. At ipsum dolores qui enim quisquam? Veniam suscipit, error consequatur excepturi adipisci maxime doloribus quidem.
+          </p>
+
+          <p>
+            <small
+              >Please contact us per Email for any further questions about
+              kuda</small
+            >
+          </p>
+          <p>
+            <small><strong>ntt.london@gmail.com</strong></small>
+          </p>
+        </div>
+        <div class="logo-description text-center">
+          <h2>Creative Commons Global Summit 2015 Logo</h2>
```

Comment:
- [ ] Kindly use `main` to order the main structure of the page instead of `div`. See my comments about semantic HTML tags below.
- [ ] The About page should contain:
-  a top section with a centralised column that includes a page title, a text block with the page description, and a line with contact information or any other prompt under it. The contact should be a link as it is an email address. It should be bold and underlined as shown in the Behance design.
- a middle section with a centralised column that includes a section header, a divider, a paragraph of text, and an image or large logo.
- a bottom section with a centralised column that includes a section header, a divider, a line of text, and two images with text centred over them. You are missing this section. Kindly add it.

_See images below._

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/123964459-46a55a80-d968-11eb-9e61-3be299ad1e43.png) | ![image](https://user-images.githubusercontent.com/49621279/123964527-5b81ee00-d968-11eb-9347-1a715bab8f1b.png)
![image](https://user-images.githubusercontent.com/49621279/123964629-78b6bc80-d968-11eb-9a5c-6e551a69ca3b.png) | ![image](https://user-images.githubusercontent.com/49621279/123964691-8f5d1380-d968-11eb-9ae3-dd9178ebf9bc.png)